src/api/app/views/webui2/webui/package/_buildstatus.html.haml

```diff
@@ -14,19 +14,20 @@
         - previous_repo = nil
         - results.each do |result|
           - repository_name = result.repository.tr('.', '_')
-          - expanded = !collapsed_repositories.include?(repository_name)
+          - package_name = package.tr('.:', '_')
+          - expanded = collapsed_repositories[package_name] ? !collapsed_repositories[package_name].include?(repository_name) : true
```

Comment:
Instead of doing `!something.include?(...)`, you can do `something.exclude?(...)`.